pythonforandroid/recipes/uvloop/__init__.py

```diff
@@ -0,0 +1,17 @@
+from pythonforandroid.recipe import PyProjectRecipe
+
+
+class UvloopRecipe(PyProjectRecipe):
+    # 0.19.0
+    version = '6c770dc3fbdd281d15c2ad46588c139696f9269c'
```

Comment:
I guess this is to make sure the tag doesn't get replaced in the upstream repo without us noticing?
I think a more common approach is to use the `sha512sum` or `md5sum`.
One advantage I'm seeing is hash verification against a version is enforced, while we don't enforce the "0.19.0" comment above the version vs the commit hash.
This is my personal take and I'm happy to get your opinion on it, but it's not blocking the merge

Reply:
My mistake, forgot to update version :sweat_smile: